packages/Button/Button.jsx

```diff
@@ -74,6 +73,15 @@ export const StyledButton = styled.button(
       color,
       '&:hover': hover,
     }
+  },
+  ({ fullwidth }) => {
+    let width
+    if (fullwidth) {
+      width = fullwidthButton
```

Comment:
Here you can add `width = '100%'` and delete the variable on L21.

Reply:
Hi @theetrain,
I can use 'width = '100%' !important' because parent defined width : auto.

Reply:
I have removed variable